Parsers/Corelight/Corelight

```diff
@@ -0,0 +1,621 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as Corelight.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. Corelight | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+let Corelight_view = view () {
+    Corelight_CL | where isnotempty(Message)
+| extend tmp = parse_json(Message)
+| evaluate bag_unpack(tmp)| extend path_parts = parse_path(log_file_s)
+    | extend EventType = extract("(^.*?)_\\d+", 1, tostring(path_parts["Filename"])),
+        EventVendor="Corelight",
+        EventProduct="Corelight Sensor",
+        SrcDvcHostname=column_ifexists('hostname_s', ''),
+        EventEndTime=column_ifexists('ts', ''),
+        SrcDvcFile=column_ifexists('log_file_s', '')
```

Comment:
sample data set has this field named 'log_file' not 'log_file_s'

Reply:
_s suffix will be added automatically for string fields. It seems to me that there is no need to add it. This log-file is a real-life example from the stdout of the LA agent.